source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -1288,24 +1420,23 @@ time during saves if your world is already fully generated.
 
 * **auto-populate**
 
-  If ``true``, newly discovered structures will be added to this config
-  with a default value of ``true``. This is useful for finding out
-  potentially what structures are being saved from various mods, and
-  allowing those structures to be selectively disabled.
+  If ``true``, newly discovered structures will be added to this config with a default value of ``true``.
+  This is useful for finding out potentially what structures are being saved from various mods,
+  and allowing those structures to be selectively disabled.
 
 | **Type:** ``boolean``
 | **Default:** ``false``
 |
 
 * **enabled**
 
-  If ``false``, disables the modification to prevent certain structures
-  from saving to the world's data folder. If you wish to prevent certain
-  structures from saving, leave this ``enabled=true``. When ``true``, the
-  modification allows for specific ``named`` structures to NOT be saved to
-  disk. Examples of some structures that are costly and somewhat irrelivent
-  is ``mineshaft``\s, as they build several structures and save, even after
-  finished generating.
+  Global switch to enable sponge's changes to the structure saving mechansim.
+  By default, this option is disabled, i.e. saving structures is not affected.
+  If you want to prevent that a certain ``named`` structure is saved to the world's folder,
```

Comment:
Change to "If you want to prevent a certain ``named`` structure being saved to the world's folder" ...
As above, so below ...